core/core/src/services/mod.rs

```diff
@@ -259,11 +259,6 @@ mod vercel_artifacts;
 #[cfg(feature = "services-vercel-artifacts")]
 pub use vercel_artifacts::*;
 
-#[cfg(feature = "services-vercel-blob")]
-mod vercel_blob;
```

Comment:
Seems code under `vercel_blob` is not removed. I didn't expect that we can see 1000 lines of changes.

Reply:
the delete is not commit, let me commit